src/pyclaw/cfl.py

```diff
@@ -22,6 +22,8 @@ def get_cached_max(self):
     def set_local_max(self,new_local_max):
         self._global_max = new_local_max
 
-    def update_global_max(self,new_local_max):
-        self._global_max = new_local_max
+    def set_global_max(self,new_global_max):
```

Comment:
Correct. I removed that method. Also ``set_local_max`` is not currently being used.